sprv/type/zif_aff_sprv_v1.intf.abap

```diff
@@ -0,0 +1,37 @@
+interface zif_aff_sprv_v1
+public .
+
+
+  types:
+    "! <p class="shorttext">Create a service provider</p>
+    "! Create a service provider
```

Comment:
Was "Create a ..." intended in front of "service provider"?

General remark: The "shorttext" shall be written in title style, the description below in sentence style.